### DISCUSSION TITLE
Is it possible to have repo specific language that can be highlighted inside of .md files?

### DISCUSSION THREAD

**Discussion by ArachnidAbby** [github-linguist/linguist#6138] [Q&A] [answered]
## Question

I was wondering if there was any way I can have a custom language grammar defined in my repo in order to use it in code blocks.

Otherwise, code for my custom language becomes hard to read inside of github. Especially for example code inside of documentation or the `readme.md`

---

### Details

the idea is that I could do something like this in my markdown files

````markdown
```mylanguagename
code in my language
```
````

Thanks in advance!

**Comment by lildude** [comment, marked as answer]
No, this is not possible. The closest you'll be able to get is to pick a language that looks similar that Linguist and GitHub already support.
